client/src/components/Explorer/ExplorerHeader.js

```diff
@@ -9,32 +9,27 @@ import Icon from '../../components/Icon/Icon';
  * The bar at the top of the explorer, displaying the current level
  * and allowing access back to the parent level.
  */
-const ExplorerHeader = ({ page, depth, onClick }) => {
-  const isRoot = depth === 1;
-
-  return (
-    <Button
-      href={page.id ? `${ADMIN_URLS.PAGES}${page.id}/` : ADMIN_URLS.PAGES}
-      className="c-explorer__header"
-      onClick={onClick}
-    >
-      <div className="c-explorer__header__inner">
-        <Icon
-          name={isRoot ? 'home' : 'arrow-left'}
-          className="icon--explorer-header"
-        />
-        <span>{page.admin_display_title || STRINGS.PAGES}</span>
-      </div>
-    </Button>
-  );
-};
+const ExplorerHeader = ({ page, onClick }) => (
+  <Button
+    href={page.id ? `${ADMIN_URLS.PAGES}${page.id}/` : ADMIN_URLS.PAGES}
+    className="c-explorer__header"
+    onClick={onClick}
+  >
+    <div className="c-explorer__header__inner">
+      <Icon
+        name={page.parent ? 'arrow-left' : 'home'}
```

Comment:
Are there any implications to this for cases where the user only has access to some sub-pages in the site’s page tree?

Reply:
Yep! IIRC, this is handled by the admin API (but I might be misremembering some other work I did in https://github.com/wagtail/wagtail/pull/5679). Been so long since I worked on any of that so will have to check

Reply:
It looks like the admin API returns the details of the `parent` page even if the user has no permissions on it, so this no longer works as it used to:

![blogger only](https://user-images.githubusercontent.com/877585/95447581-d7ab5000-0959-11eb-8619-e5668e1a3b7f.png)

This is with a user who only has access to the Blog page in the bakerydemo. Ideally it should show the Home icon, and more importantly this shouldn’t be clickable (causes a JS error currently). I think this might be fixable simply enough by basing this on `depth`, so it’s relative to the user’s navigation, wherever it happens to start from.

Reply:
Ahh, I think I changed this logic in https://github.com/wagtail/wagtail/pull/5679 but forgot that isn't merged yet. Thanks for spotting!